auto_pano.sh

```diff
@@ -14,40 +14,40 @@ done
 shift $((OPTIND-1))
 [ "$1" = "--" ] && shift
 
-DIR=$1
+DIR="$1"
 
-if [ x$DIR == x"" ]; then
+if [ x"$DIR" == x"" ]; then
     echo >&2 "Usage: auto_pano.sh [-s] [-h] dir"
     echo >&2 "    No directory provided"
     exit 2
 fi
-if [ ! -d $DIR ]; then
+if [ ! -d "$DIR" ]; then
     echo "Directory $DIR does not exist."
     exit 3
 fi
 echo "Processing directory $DIR"
 
 # generate the project name from the first input file name
-BASE_NAME=$(ls $DIR/*.JPG | head -n 1 | sed -e 's/\.JPG$/_pano/')
-OUTPUT_FILE=$BASE_NAME'.pto'
+BASE_NAME=$(ls "$DIR"/*.JPG | head -n 1 | sed -e 's/\.JPG$/_pano/')
+OUTPUT_FILE="$BASE_NAME"'.pto'
 echo "creating '$OUTPUT_FILE'..."
 
 # Pass the stack size to pto_gen if needed
 if [ x$STACK_SIZE == x"" ]; then
-    pto_gen $DIR/*.JPG -o $OUTPUT_FILE
+    pto_gen "$DIR/"*.JPG -o "$OUTPUT_FILE"
 else
-    pto_gen $DIR/*.JPG -s $STACK_SIZE -o $OUTPUT_FILE
+    pto_gen "$DIR/"*.JPG -s $STACK_SIZE -o "$OUTPUT_FILE"
 fi
 
-cpfind -o $OUTPUT_FILE $OUTPUT_FILE
-#cpclean -o $OUTPUT_FILE $OUTPUT_FILE
-autooptimiser -a -l -s -o $OUTPUT_FILE $OUTPUT_FILE
+cpfind -o "$OUTPUT_FILE" "$OUTPUT_FILE"
+#cpclean -o "$OUTPUT_FILE" "$OUTPUT_FILE"
+autooptimiser -a -l -s -o "$OUTPUT_FILE" "$OUTPUT_FILE"
 
 # -p 0 => rectaliniar projection
-#pano_modify -o $OUTPUT_FILE -p 0 --ldr-file=JPG --output-type=BF $OUTPUT_FILE
-pano_modify -o $OUTPUT_FILE --output-type=BF $OUTPUT_FILE
-pano_modify -o $OUTPUT_FILE --center --straighten --canvas=AUTO --crop=AUTO $OUTPUT_FILE
+#pano_modify -o "$OUTPUT_FILE" -p 0 --ldr-file=JPG --output-type=BF "$OUTPUT_FILE"
+pano_modify -o "$OUTPUT_FILE" --ldr-file=JPG --ldr-compression=95 --output-type=BF "$OUTPUT_FILE"
```

Comment:
mind adding an option for this? I prefer tif for my panos, as I usually do some post-processing and want the full depth. It might make sense to add an option to the HDR one too, but I personally would leave that one at JPG so it's not really needed.